book/src/01_getting_started/01_setup.md

```diff
@@ -25,5 +25,6 @@ If you're using the [Homebrew](https://brew.sh/) package manager, run `brew bund
 If you would like to set up the environment manually, you will need to install the following dependencies:
 
 - Install Rust using [rustup](https://www.rust-lang.org/tools/install)
+- Install `cargo hack`: `cargo install cargo-hack`
```

Comment:
Might be unnecessary

Reply:
Removed in [0cb0dc8](https://github.com/NomicFoundation/hardhat/pull/4380/commits/0cb0dc81d17d19c1f20b00fc0918d2451b727e20)